src/activities/quizzes/attempts/QuizAttemptsEntity.js

```diff
@@ -106,6 +127,32 @@ export class QuizAttemptsEntity extends Entity {
 		return this._entity && this._entity.getSubEntityByClass(Classes.quizzes.attempts.retakeIncorrectOnly);
 	}
 
+	/**
+	 * @returns {object} quiz attempt conditions sub-entity
+	 */
+	getAttemptConditionsSubEntity() {
+		if (this._entity.hasClass(Classes.quizzes.attempts.attemptConditions) && this._entity.hasClass('collection')) {
+			return this._entity && this._entity.getSubEntityByClass(Classes.quizzes.attempts.attemptConditions);
+		}
+
+		return null;
+	}
+
+	/**
+	 * @returns {object} a singular quiz attempt condition sub-entity
+	 */
+	getAttemptConditionSubEntity(attemptConditionNumber) {
+		entity = this.getAttemptConditionsSubEntity();

```

Comment:
I can't see where these variables are declared. Are we overriding global vars? for `entity` and `attemptConditionEntities`?